ci/workflows/main-cron.yml

```diff
@@ -33,15 +33,25 @@ steps:
     command: "ci/scripts/compute-node-test.sh"
     plugins:
       - seek-oss/aws-sm#v2.3.1:
-            env:
-              CODECOV_TOKEN: my-codecov-token
+          env:
+            CODECOV_TOKEN: my-codecov-token
```

Comment:
does this token really work? cc @huangjw806

Reply:
This token is the real work, it is stored in AWS Secrets Manager.

Reply:
That's cool 🤣 So we recorded a mapping between `my-codecov-token` and the real token?

Reply:
Yes, this way we can ensure our secrets are safe.